### DISCUSSION TITLE
I was gettingValidationError: "navbar.items[1].items[0].position, while trying to use the Dropdown feature

### DISCUSSION THREAD

**Discussion by bamideleLAWAL** [facebook/docusaurus#5920] [Q&A] [answered]
I was getting this error:-  ValidationError: "navbar.items[1].items[0].position is not allowed,  while trying to use the drop-down feature as documented in the official docs. Really, i was having issue with the drop-down stuff. Need help please!

**Comment by slorber** [comment, marked as answer]
The position field is only meant to be used for top-level navbar items, not items nested inside the dropdown
